core/logger/logger.go

```diff
@@ -197,6 +197,7 @@ func (c *Config) New() (Logger, func() error) {
 
 	l = newSentryLogger(l)
 	l = newPrometheusLogger(l)
+	// l = newBeholderLogger(l)
```

Comment:
could hook in as early as here but seems excessive (all overloaded funcs will emit a custom message) for this first use case of integrating with Keystone